cwms-data-api/src/test/java/cwms/cda/api/location/kind/OutletControllerTestIT.java

```diff
@@ -188,6 +206,121 @@ void test_outlet_rename() {
         ;
     }
 
+    @Test
+    void test_rating_spec_id() {
```

Comment:
Let's go over the design of this.  We need to have two unit tests for controlled and uncontrolled outlets.